### PR TITLE
Converted to new mask format

### PR DESCRIPTION
**Pull request by jfcrenshaw**
Converted to new mask format and added `thetaMax`

### CODE REVIEW

File: danish/factory.py [hunk 1]
```diff
@@ -648,34 +648,47 @@ class DonutFactory:
...
+        Nested dictionary containing the mask model. See the notes below
+        for details on the format.
     focal_length : float
         Focal length in meters.
     pixel_scale : float
         Pixel scale in meters.
+
+    Notes
+    -----
+    The mask_params dictionary is a nested dictionary the specifies the
```
Comment (jmeyers314) [inline danish/factory.py:661]:
```suggestion
    The mask_params dictionary is a nested dictionary that specifies the
```

**Review by jmeyers314 — APPROVED** [review]
LGTM.  Thanks!

**Comment by jfcrenshaw** [comment]
@jmeyers314 I fixed the typo in the docs. Looks like everything is ready, but you have to do the merge because I don't have permissions

**Comment by jmeyers314** [comment]
Merged.  Thanks!

**Pull request merged**